src/Dfc.CourseDirectory.Database/Pttcd/Views/vwProviderSearchIndex.sql

```diff
@@ -3,11 +3,19 @@ AS
 SELECT      p.ProviderId,
             p.Ukprn,
             p.ProviderName,
+            ISNULL(pc.AddressPostTown, pc.AddressItems) Town,
+            pc.AddressPostcode Postcode,
             p.ProviderStatus,
             p.UkrlpProviderStatusDescription,
-            pc.AddressPostcode Postcode,
-            ISNULL(pc.AddressPostTown, pc.AddressItems) Town,
-            p.Version -- Timestamp (RowVersion) type, used here for cosmos HighWaterMark
+            p.Version, -- Timestamp (RowVersion) type, used here for cosmos HighWaterMark
+
+            -- additional fields to skew the search index score approximately as per the old index
+            coalesce(p.CourseDirectoryName,p.ProviderName) CourseDirectoryName, -- as per previous cosmos-based search index definition
+            p.TradingName,
+            p.Alias,
+            p.UpdatedOn,
+            cast(null as datetime) as DateOnboarded,
```

Comment:
Use CAPS for function names and `NULL` for readability

Reply:
I will reformat the file for consistency with everything else we have as suggested.

As an aside, personally I'd drop the all-caps use of sql across the board having stopped to think about why it's like that in the first place and how that affects readability. It's not required by the parser (as of sql-99). It's just a hangover from a language that was created in 1974 and everyone is just so used to reading and writing it that way that it's the norm. After finding it a bit jarring the first few times I did it all in lowercase I very quickly started to find it easier to read.

Related:

* https://stackoverflow.com/questions/9426138/why-are-the-queries-in-sql-mostly-written-in-capital-letters
* https://www.mity.com.au/blog/writing-readable-content-and-why-all-caps-is-so-hard-to-read